VLC-iOS-Bridging-Header.h

```diff
@@ -7,6 +7,7 @@
 #import "VLCPlaylistCollectionViewCell.h"
 #import "VLCMediaDataSource.h"
 #import <PAPasscode/PAPasscodeViewController.h>
+#import <MobileVLCKit/MobileVLCKit.h>
```

Comment:
`VLCDisplayController, VLCMiniPlaybackView: first playback fixes` commit description seems to not match at all what is done in the commit?

Reply:
I think this is due to merging. I will fix this commit since it's only adding this file in the header